app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Remove these files 
![image](https://user-images.githubusercontent.com/54136347/127820054-5c9eca99-df3b-4d56-9eef-bb5be5283cdb.png)
Can't write it on them because they are empty